migrate/versions/1f5853959c65_init_db.py

```diff
@@ -17,10 +17,97 @@
 depends_on = None
 
 
+# Alembic 1.9.4 does not support dropping enums on downgrade on autogen. So,
+# ... we separate enum declarations from upgrade.
+def drop_enums(*enums: sa.Enum):
```

Comment:
Not a major deal, but I think it's a bit more "pythonic" to pass in an iterable instead of using *args when passing in a set of items to be acted against.